frontend/app/assets/javascripts/spree/frontend/checkout/coupon-code.js.coffee

```diff
@@ -0,0 +1,39 @@
+Spree.onCouponCodeApply = (e) ->
+  coupon_code_field = $('#order_coupon_code')
+  coupon_code = $.trim(coupon_code_field.val())
+  return if coupon_code == ''
+
+  if $('#coupon_status').length == 0
+    coupon_status = $("<div id='coupon_status'></div>")
+    coupon_code_field.parent().append(coupon_status)
+  else
+    coupon_status = $("#coupon_status")
+
+  successClass = coupon_status.data('success-class') || 'success'
+  errorClass = coupon_status.data('error-class') || 'error'
```

Comment:
From my experience in integrating this code with Foundation, it was easier to specify the classes as data attributes than to add additional CSS to match this specific use case. I'd always avoid extra CSS if possible.

Unless others favour your opinion, I'd prefer to leave the data attributes in place.